scripts/imgtool/main.py

```diff
@@ -138,14 +143,45 @@ def getpub(key, encoding, lang):
         # `default=valid_encodings[0]` should be added to `-e` param.
         lang = valid_langs[0]
     key = load_key(key)
+
+    if not output:
+        output = sys.stdout
     if key is None:
         print("Invalid passphrase")
     elif lang == 'c' or encoding == 'lang-c':
-        key.emit_c_public()
+        key.emit_c_public(file=output)
     elif lang == 'rust' or encoding == 'lang-rust':
-        key.emit_rust_public()
+        key.emit_rust_public(file=output)
     elif encoding == 'pem':
-        key.emit_public_pem()
+        key.emit_public_pem(file=output)
+    elif encoding == 'raw':
+        key.emit_raw_public(file=output)
+    else:
+        raise click.UsageError()
+
+
+@click.option('-e', '--encoding', metavar='encoding',
+              type=click.Choice(valid_hash_encodings),
+              help='Valid encodings: {}. '
+                    'Default value is {}.'.format(', '.join(valid_hash_encodings), valid_hash_encodings[0]))
```

Comment:
Please break it into multiple lines, checking against PEP8 style guide gives:
`scripts/imgtool/main.py:166:80: E501 line too long (108 > 79 characters)`

Reply:
Done